integ-test/src/test/java/org/opensearch/sql/sql/ConvertTZFunctionIT.java

```diff
@@ -0,0 +1,151 @@
+  }
+
+  @Test
+  public void inRangeSameTimeZone() throws IOException {
+    var result = executeJdbcRequest(
+        "SELECT convert_tz('2021-05-12 11:34:50','+09:00','+09:00')");
+    verifySchema(result,
+        schema("convert_tz('2021-05-12 11:34:50','+09:00','+09:00')", null, "datetime"));
+    verifyDataRows(result, rows("2021-05-12 11:34:50"));
+  }
+
+  @Test
+  public void inRangeTwentyFourHourTimeOffset() throws IOException {
+    var result = executeJdbcRequest(
+        "SELECT convert_tz('2021-05-12 11:34:50','-12:00','+12:00')");
+    verifySchema(result,
+        schema("convert_tz('2021-05-12 11:34:50','-12:00','+12:00')", null, "datetime"));
+    verifyDataRows(result, rows("2021-05-13 11:34:50"));
+  }
+
+  @Test
+  public void inRangeFifteenMinuteTimeZones() throws IOException {
+    var result = executeJdbcRequest(
+        "SELECT convert_tz('2021-05-12 13:00:00','+09:30','+05:45')");
+    verifySchema(result,
+        schema("convert_tz('2021-05-12 13:00:00','+09:30','+05:45')", null, "datetime"));
+    verifyDataRows(result, rows("2021-05-12 09:15:00"));
+  }
+
+  @Test
+  public void inRangeRandomTimes() throws IOException {
+    var result = executeJdbcRequest(
+        "SELECT convert_tz('2021-05-12 13:00:00','+09:31','+05:11')");
+    verifySchema(result,
+        schema("convert_tz('2021-05-12 13:00:00','+09:31','+05:11')", null, "datetime"));
+    verifyDataRows(result, rows("2021-05-12 08:40:00"));
+  }
+
+  @Test
+  public void nullToFieldUnder() throws IOException {
+    var result = executeJdbcRequest(
+        "SELECT convert_tz('2021-05-30 11:34:50','-14:00','+08:00')");
+    verifySchema(result,
+        schema("convert_tz('2021-05-30 11:34:50','-14:00','+08:00')", null, "datetime"));
+    verifyDataRows(result, rows(new Object[]{null}));
+  }
+
+  @Test
+  public void nullFromFieldOver() throws IOException {
+    var result = executeJdbcRequest(
+        "SELECT convert_tz('2021-05-12 11:34:50','-12:00','+14:01')");
+    verifySchema(result,
+        schema("convert_tz('2021-05-12 11:34:50','-12:00','+14:01')", null, "datetime"));
+    verifyDataRows(result, rows(new Object[]{null}));
+  }
+
+  @Test
+  public void inRangeMinOnPoint() throws IOException {
+    var result = executeJdbcRequest(
+        "SELECT convert_tz('2021-05-12 15:00:00','-13:59','-13:59')");
+    verifySchema(result,
+        schema("convert_tz('2021-05-12 15:00:00','-13:59','-13:59')", null, "datetime"));
+    verifyDataRows(result, rows("2021-05-12 15:00:00"));
+  }
+
+  @Test
+  public void nullFromGarbageInput() throws IOException {
+    var result = executeJdbcRequest(
+        "SELECT convert_tz('2021-05-12 11:34:50','****','+14:01')");
+    verifySchema(result,
+        schema("convert_tz('2021-05-12 11:34:50','****','+14:01')", null, "datetime"));
+    verifyDataRows(result, rows(new Object[]{null}));
+  }
+
+  @Test
+  public void nullFromGarbageInput2() throws IOException {
```

Comment:
What's garbage here?

Reply:
Added a comment in code. Also changed the function name to use `invalid` instead of `garbage`. Invalid input would be anything not matching the timezone format for the two timezone fields:
`(+/-)HH:mm`
Disregard the brackets, it just means pick either `+` or `-`.